### ISSUE TITLE
v1.3.0 is unable to build from source on windows via cargo install+ crates.io with rust 1.67.1 stable

### ISSUE THREAD

**Issue by bruxisma** [solidiquis/erdtree#50]
Hello!

Thanks for continuing to improve this tool It's one of my favorites 🙂 

After trying to upgrade to `erdtree` 1.3.0 with `cargo install` via crates.io, I got the following error output (placed inside a `<details>` block for brevity, with truncated path)

<details>

```console
error[E0425]: cannot find value `metadata` in this scope
  --> ${HOME}\.cargo\registry\src\github.com-1ecc6299db9ec823\erdtree-1.3.0\src\fs\inode.rs:54:13
   |
54 |             metadata.file_index(),
   |             ^^^^^^^^ not found in this scope
   |
help: consider importing one of these items
   |
1  | use core::ptr::metadata;
   |
1  | use std::fs::metadata;
   |
1  | use std::ptr::metadata;
   |

error[E0425]: cannot find value `metadata` in this scope
  --> ${HOME}\.cargo\registry\src\github.com-1ecc6299db9ec823\erdtree-1.3.0\src\fs\inode.rs:55:13
   |
55 |             metadata.volume_serial_number(),
   |             ^^^^^^^^ not found in this scope
   |
help: consider importing one of these items
   |
1  | use core::ptr::metadata;
   |
1  | use std::fs::metadata;
   |
1  | use std::ptr::metadata;
   |

error[E0425]: cannot find value `metadata` in this scope
  --> ${HOME}\.cargo\registry\src\github.com-1ecc6299db9ec823\erdtree-1.3.0\src\fs\inode.rs:56:13
   |
56 |             metadata.number_of_links(),
   |             ^^^^^^^^ not found in this scope
   |
help: consider importing one of these items
   |
1  | use core::ptr::metadata;
   |
1  | use std::fs::metadata;
   |
1  | use std::ptr::metadata;
   |

error[E0425]: cannot find value `nlink` in this scope
  --> ${HOME}\.cargo\registry\src\github.com-1ecc6299db9ec823\erdtree-1.3.0\src\fs\inode.rs:58:42
   |
58 |             return Ok(Self::new(md, dev, nlink));
   |                                          ^^^^^
   |                                          |
   |                                          a field by this name exists in `Self`
   |                                          help: a local variable with a similar name exists: `nlinks`

error[E0308]: mismatched types
  --> ${HOME}\.cargo\registry\src\github.com-1ecc6299db9ec823\erdtree-1.3.0\src\fs\inode.rs:58:33
   |
58 |             return Ok(Self::new(md, dev, nlink));
   |                       --------- ^^ expected `u64`, found struct `Metadata`
   |                       |
   |                       arguments to this function are incorrect
   |
note: associated function defined here
  --> ${HOME}\.cargo\registry\src\github.com-1ecc6299db9ec823\erdtree-1.3.0\src\fs\inode.rs:18:12
   |
18 |     pub fn new(ino: u64, dev: u64, nlink: u64) -> Self {
   |            ^^^ --------

Some errors have detailed explanations: E0308, E0425.
For more information about an error, try `rustc --explain E0308`.
error: could not compile `erdtree` due to 5 previous errors
error: failed to compile `erdtree v1.3.0`, intermediate artifacts can be found at `${HOME}\AppData\Local\Temp\cargo-installK25LHL`
```

</details>

To replicate this, I simply use `cargo install erdtree`

**Comment by solidiquis** [comment]
Ahh yes my bad. I'll have a minor version bump tonight and resolve this issue! Thank you for bringing this to my attention!

**Comment by solidiquis** [comment]
Hello unfortunately because I'm actually quite behind on work I'll have to push the fix to later in the week or perhaps the weekend, but the next minor version release will definitely be out before the following monday!

**Comment by bruxisma** [comment]
No worries! I can stick it out with 1.2 in the meantime 🙂

**Comment by solidiquis** [comment]
kk the issue should be resolved! [v1.4.0 is now on crates.io](https://crates.io/crates/erdtree). Checkout the release notes for the new goodies and please let me know if you're still facing issues. Sorry about that!

**Comment by bruxisma** [comment]
Confirmed as working! Thanks a bunch!! 😄
